command/config/config.go

```diff
@@ -279,14 +279,15 @@ type EnvConfig struct {
 	}
 
 	Settings struct {
-		LiteEnginePath string `envconfig:"DRONE_LITE_ENGINE_PATH" default:"https://github.com/harness/lite-engine/releases/download/v0.4.3/"`
-		DefaultDriver  string `envconfig:"DRONE_DEFAULT_DRIVER" default:"amazon"`
-		ReusePool      bool   `envconfig:"DRONE_REUSE_POOL" default:"false"`
-		BusyMaxAge     int64  `envconfig:"DRONE_SETTINGS_BUSY_MAX_AGE" default:"24"`
-		FreeMaxAge     int64  `envconfig:"DRONE_SETTINGS_FREE_MAX_AGE" default:"720"`
-		MinPoolSize    int    `envconfig:"DRONE_MIN_POOL_SIZE" default:"1"`
-		MaxPoolSize    int    `envconfig:"DRONE_MAX_POOL_SIZE" default:"2"`
-		EnableAutoPool bool   `envconfig:"DRONE_ENABLE_AUTO_POOL" default:"false"`
+		LiteEnginePath     string `envconfig:"DRONE_LITE_ENGINE_PATH" default:"https://github.com/harness/lite-engine/releases/download/v0.4.3/"`
+		DefaultDriver      string `envconfig:"DRONE_DEFAULT_DRIVER" default:"amazon"`
+		ReusePool          bool   `envconfig:"DRONE_REUSE_POOL" default:"false"`
+		BusyMaxAge         int64  `envconfig:"DRONE_SETTINGS_BUSY_MAX_AGE" default:"24"`
+		FreeMaxAge         int64  `envconfig:"DRONE_SETTINGS_FREE_MAX_AGE" default:"720"`
+		MinPoolSize        int    `envconfig:"DRONE_MIN_POOL_SIZE" default:"1"`
+		MaxPoolSize        int    `envconfig:"DRONE_MAX_POOL_SIZE" default:"2"`
+		EnableAutoPool     bool   `envconfig:"DRONE_ENABLE_AUTO_POOL" default:"false"`
+		HarnessDownloadURI string `envconfig:"DRONE_HARNESS_DOWNLOAD_URI"`
```

Comment:
Set a default similar to how we do it with lite-engine

Reply:
Also, HarnessDownloadURI tells me nothing. what's it downloading? 'HarnessTestBinaryURI'

Reply:
Ack. Will replace HarnessDownloadURI with HarnessTestBinaryURI

Reply:
@eoinmcafee00 done